docs/governance.md

```diff
@@ -0,0 +1,55 @@
+
+## Roles and Responsibilities
+
+### Users
+
+Users are community members who use projects that implement the ESTree specification. Users may become involved with the project and may find themselves becoming Contributors.
+
+### Implementers
+
+Implementers are community members who create or maintain a project that implements the ESTree specification, either through the creation or usage of data structures following the specification. Implementers who continue to engage with the project and its community will often become more and more involved. Such Implementers may find themselves becoming Contributors.
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the project, most often in the form of code and/or documentation. Anyone can become a Contributor, and contributions can take many forms. There is no expectation of commitment to the project, no specific skill requirements, and no selection process.
+
+Contributors have read-only access to the ESTree GitHub repository and so submit changes via pull requests. Contributor pull requests have their contribution reviewed and merged by an ESC member. ESC members work with Contributors to review their submission and prepare it for merging.
+
+### Member Projects
+
+Member Projects are projects that implement the ESTree specification and have donated their time to the maintenance and development of the ESTree project. Member Projects may be added or removed through a standard ESC motion (discussed in the next section).
+
+### ESTree Steering Committee (ESC)
+
+The ESLint project is jointly governed by the ESTree Steering Committee (ESC) which is responsible for high-level guidance of the project. The ESC is made up of one representative from each Member Project, and that one representative represents their Member Project's vote in any decision. It's the responsibility of the Member Project representative to drive consensus within the Member Project when decisions about the ESTree project are being made. 
+
+The ESC has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+
+ESC seats are not time-limited. The size of the ESC cannot be larger than five members. This size ensures adequate coverage of important areas of expertise balanced with the ability to make decisions efficiently.
+
+The ESC may add additional members to the ESC, up to the previously mentioned maximum, by a standard ESC motion.
+
+An ESC member may be removed from the ESC by voluntary resignation, and a Member Project may replace its ESC member at any time by informing the other ESC members.
+
+## Consensus Seeking Process
+
+The ESC follows a
```

Comment:
I think I'd still want us to explicitly mention and link to the Philosophy section from the README, since it's an important part of the decision making process.

Given how detailed this document already is, it makes little sense to leave one sentence out for a separate PR IMO.

Reply:
To me, the philosophy is relevant to inclusion of new syntax, but is not relevant for other decisions. To that end, I'd rather have a separate procedures doc that links to philosophy as part of that procedure, and also lists other procedures and their relevant philosophies as well.

Reply:
Hmm, what kind of other decisions do you have in mind? Since the spec is purely additive in nature, almost all the decisions are about new syntax. If this document doesn't cover it, that's fine to leave for a separate one, but then I'd like to understand what other parts of the process this document does cover.

Reply:
@nzakas Can you add some clarification so that we'd merge this please?

Reply:
I think there's plenty of other types of decisions to document:

1. Expanding or reducing the scope of the project
2. Add or remove new repositories (what do we do with `formal`?)
3. Whether or not to document an extension (JSX, for example)
4. What should the structure of the spec be (do we still want a bunch of mini specs or one big one?)?
5. Processes around issues and PRs (do we always wait for everyone to reply? Do we wait with a time limit? What if no one replies?)
6. The full lifecycle of adding new syntax (does it start with an issue? start with a PR to the experimental folder?)

Again, I'm not saying defining the process and philosophy for adding new syntax shouldn't be documented, I'm just saying let's do a separate procedures document so we can get this merged and continue to iterate on the procedures.

Reply:
Okay, sounds good, as long as we've clarified that this doesn't override decision process for new syntax, I don't have any objections.